cmd/codecrafters/main.go

```diff
@@ -23,11 +23,16 @@ USAGE
   $ codecrafters [command]
 
 EXAMPLES
-  $ codecrafters test     # Run tests without committing changes
-  $ codecrafters submit   # Commit changes & submit to move to next step
+  $ codecrafters test              # Run tests without committing changes
+  $ codecrafters test --previous   # Run tests for the current stage and all previous stages in ascending order
```

Comment:
```suggestion
  $ codecrafters test --previous   # Run tests for previous stages without committing changes
```

Let's just say this for now - in the future we'll change the actual behaviour to match this.